cudamapper/src/application_parameters.cpp

```diff
@@ -219,6 +222,27 @@ void ApplicationParameters::create_input_parsers(std::shared_ptr<io::FastaParser
         target_parser = io::create_kseq_fasta_parser(target_filepath, kmer_size + windows_size - 1);
     }
 
+    number_of_basepairs_t total_sequence_length                 = 0;
+    const number_of_basepairs_t minimum_for_automatic_filtering = 500000; // Require at least 0.5Mbp of sequence for filtering by default
```

Comment:
Do we want to have this condition for query and target together or separate?
What happens if we have a very long query, but the target only has two reads?
(not sure about this, just asking)

Reply:
Agreed, though this isn't a case we target right now (like read-ref mapping).

Reply:
Ok, marking as resolved